.travis.yml

```diff
@@ -40,6 +40,9 @@ matrix:
       env: STATIC_DEPS=false
     - python: pypy3
       env: STATIC_DEPS=false
+    - python: 3.8
+      env: STATIC_DEPS=false
```

Comment:
Why (only) this combination?

Reply:
For no particular reason.
I didn't want to flood the matrix with arm64 jobs (which are not really fast right now) so i decided to pick only one combination. 
Now I've added a job with `STATIC_DEPS=true`, but if You think it's worth to fully test the architecture i can add all the required combinations